src/bundle/Resources/public/scss/_field-group.scss

```diff
@@ -93,6 +93,18 @@
             background-color: $ez-white;
             padding: calculateRem(16px) calculateRem(24px);
             border-radius: 0 0 calculateRem(5px) calculateRem(5px);
+
+            .ezstring-field {
+                display: inline-block;
+                max-width: 100%;
+                overflow: hidden;
+                text-overflow: ellipsis;
+            }
+
+            .ezrichtext-field p {
```

Comment:
We should avoid styling by tag name also this will solve the problem only for the paragraph what about headings and other stuff?